web/src/App.tsx

```diff
@@ -105,10 +104,7 @@ export default function App(): ReactElement {
                 <TimeControllerWrapper />
               </Suspense>
               <Suspense>
-                <FeatureFlagsManager />
-              </Suspense>
-              <Suspense>
-                <LegendContainer />
+                <MapCardContainer />
```

Comment:
I think the naming here is a bit confusing, how about "MapOverlays" or something like that?